recipe/meta.yaml

```diff
@@ -18,11 +18,11 @@ requirements:
     build:
         - python  # [win]
         - cmake
-        - libtiff 4.0.6
+        - libtiff 4.0.*
```

Comment:
Ah, OK. Thanks @ocefpaf. I'll take a look. If that is the case, we will need to change the pinning scheme.